tests/Resources/Page/ODH/JupyterHub/JupyterHubSpawner.robot

```diff
@@ -142,6 +142,16 @@ Spawn Notebook With Arguments
       END
       Wait Until Element is Visible  id:progress-bar
       Run Keyword And Continue On Failure  Wait Until Page Does Not Contain Element  id:progress-bar  ${spawner_timeout}
+      Wait for JupyterLab Splash Screen  timeout=30
+      Maybe Close Popup
+      ${is_launcher_selected} =  Run Keyword And Return Status  JupyterLab Launcher Tab Is Selected
+      Run Keyword If  not ${is_launcher_selected}  Open JupyterLab Launcher
+      Open With JupyterLab Menu  File  New  Notebook
```

Comment:
I think you may have to add sleep/wait time after opening the notebook. Sometime it take time for pop to appear on the screen.